rlgear/utils.py

```diff
@@ -262,6 +262,21 @@ def shorten_dfs(_dfs: Sequence[pd.DataFrame], max_step: int = None) -> None:
         _dfs[i] = _df[_df.index <= max_step]  # type: ignore
 
 
+def preprocess_pbt_df(df: pd.DataFrame, x_tag: str) -> pd.DataFrame:
+    tag_diff = '{}_diff'.format(x_tag)
+    df[tag_diff] = df[x_tag].diff()
+    neg_diff = df.loc[df[tag_diff] < 0].index.to_list()
+    if len(neg_diff) == 0:
+        return df
+    else:
+        for _, val in enumerate(neg_diff):
+            start = val
+            end = len(df)
+            df[x_tag][start:end] += df[tag_diff][start-2] +\
```

Comment:
can we add a space between the `+` and `\`. Obviously it isn't a big deal; this just matches the rest of the code.

Reply:
made this edit